system/emergency_goal_manager/include/emergency_goal_manager_core.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Tier IV, Inc.
```

Comment:
The copyright year is 2022. Is there a original file for this?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/5634/commits/10723f4cd1ae6fe7257a626e10ff991232327ac0